packages/cli/test/integration/credentials/credentials.api.test.ts

```diff
@@ -9,14 +9,14 @@ import { ProjectRepository } from '@db/repositories/project.repository';
 import type { Project } from '@db/entities/Project';
 import { CredentialsRepository } from '@db/repositories/credentials.repository';
 import { SharedCredentialsRepository } from '@db/repositories/sharedCredentials.repository';
+import { randomString } from '@/utils/random';
```

Comment:
Why not re-export it from `../shared/random`?

Reply:
we could do that, but what's the advantage?